utils.py

```diff
@@ -666,14 +736,16 @@ def require_valid_url_fragment(name, name_type, allowed_length):
             '%s field should not exceed %d characters, '
             'received %s.' % (name_type, allowed_length, name))
 
-    if not re.match(constants.VALID_URL_FRAGMENT_REGEX, name):
+    if not re.match(
+            constants.VALID_URL_FRAGMENT_REGEX, name): # type: ignore[attr-defined] # pylint: disable=line-too-long
```

Comment:
You have to clearly define this approach for dealing with line limit exceeded error due to type ignore in the wiki page. Once you added details there I will resolve this.

Reply:
Added to the wiki. Thanks!